michelson.md

```diff
@@ -323,13 +326,13 @@ The following unit test groups are not supported by the symbolic interpreter.
 ```k
   syntax KItem ::= "#ConvertParamToNative"
   rule <k> #ConvertParamToNative => .K ... </k>
-       <paramvalue> D:Data => #MichelineToNative(D, #ConvertToType(T), .Map, BigMaps) </paramvalue>
-       <paramtype>  T      => #ConvertToType(T)                                       </paramtype>
+       <paramvalue> D:Data => #MichelineToNative(D, { #AddDefaultEntry(TypeMap, #Type(unit)) [ %default ] }:>Type, .Map, BigMaps) </paramvalue>
+       <paramtype> TypeMap => #AddDefaultEntry(TypeMap, #Type(unit)) </paramtype>
```

Comment:
Check if you can remove the `#AddDefaultEntry` and see if tests pass

Reply:
This is needed because sometimes this code runs before the local entrypoint map can be initialized.